### PR TITLE
Fix github container registry

### PR DESCRIPTION
**Pull request by maruina**
It seems I was following the wrong tutorial.
- Fix the login steps as per https://github.com/docker/login-action#github-container-registry
- ~Created a PAT as per https://docs.github.com/en/packages/guides/migrating-to-github-container-registry-for-docker-images#authenticating-with-the-container-registry~
- Push action changed as per https://github.com/docker/build-push-action#push-to-multi-registries

### CODE REVIEW

File: .github/workflows/release-artifacts.yaml [hunk 1]
```diff
@@ -25,12 +25,15 @@ jobs:
...
       id: tagName
 
-    - name: Build on PR
-      id: docker_build
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.CR_PAT }}
```
Comment (sledigabel) [inline .github/workflows/release-artifacts.yaml:33]: Looks like this is something we need to create manually this time, I assume you've done this already? Is this a company-shared secret?

Comment (sledigabel) [inline .github/workflows/release-artifacts.yaml:33]: we prob don't need to share the PAT anywhere, and will invalidate/regenerate if we need?

**Review by sledigabel — APPROVED** [review]
looks nice and clean; fixed the github package auth and push part

File: .github/workflows/release-artifacts.yaml (hunk 1 above)
Comment (maruina) [reply .github/workflows/release-artifacts.yaml:33]: Yes, if we need a new one any owner can do it.

**Comment by maruina** [comment]
Pending enabling `GitHub Container Registry` for our organization.

**Comment by maruina** [comment]
Had a chat with @porelle-logan and he advised to stick to the "old" GitHub Packages Docker registry.

**Review by sledigabel — APPROVED** [review]
LGTM

File: .github/workflows/release-artifacts.yaml (hunk 1 above)
Comment (maruina) [reply .github/workflows/release-artifacts.yaml]: And now we don't need that until we migrate to the new registry.

**Comment by maruina** [comment]
And it's working :)
https://github.com/Skyscanner/argocd-progressive-rollout/packages/618978

**Pull request merged**